src/api/video/video.controller.js

```diff
@@ -0,0 +1,41 @@
+const Videos = require('./video.model');
+
+async function getVideos(req, res, next) {
+  try {
+    let response;
+    if ('id' in req.params) {
+      if (!/^[a-fA-F0-9]{24}$/.test(req.params.id)) throw new ReferenceError('Invalid Video ID.');
+      try {
+        response = [await Videos.findById(req.params.id).exec()];
+        if (response[0] === null) throw new ReferenceError('The requested ID does not exist.');
+      } catch ({ message }) {
+        throw new ReferenceError(message);
+      }
+    } else response = await Videos.find({}).exec();
+
+    const finalResponse = response.map(({
+      id, type, name, videoID, title, date, description, url, thumbnail, createdAt, updatedAt,
```

Comment:
I don't like this formatting looks as is (not your fault). The pending PR #49 addresses this with the prettier implementation. We should wait to merge this PR until #49 is merged.

Reply:
I agree that this format isn't ideal, but let's not let this hold up progression until we've delivered the MVP

Reply:
We aren't in a rush on these items as the frontend team is still working on their basic framework.